apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnection.cs

```diff
@@ -261,6 +274,104 @@ public SpannerTransaction BeginReadOnlyTransaction(TransactionId transactionId)
         public Task<SpannerTransaction> BeginTransactionAsync(CancellationToken cancellationToken = default) =>
             BeginTransactionImplAsync(s_readWriteTransactionOptions, TransactionMode.ReadWrite, cancellationToken);
 
+        /// <summary>
+        /// Executes a read-write transaction, with retries as necessary.
```

Comment:
Possibly clarify in "remarks" what's meant by "as necessary"? (It's only retrying certain things.)